GPT/gpt-function-calling.talon

```diff
@@ -0,0 +1,15 @@
+mode: command
+tag: user.cursorless
```

Comment:
This scopes all the go commands to just cursorless. Is that what you want?  I wasn't sure if you want to only function call within visual studio code at the moment.

Reply:
One of the commands requires cursorless, so I wasn't sure what to do there.  I didn't really want to split out a separate file. What do you think?

Reply:
I would split it. No reason the other command shouldn't be available outside of cursorless. We could have one `.talon` file where we put all the cursorless stuff if you wanna avoid lots of small files. For that matter, I'm not sure the other command in this file needs its own file

Reply:
👍 Sounds good to me. I pushed these changes.